bootstrap.sh

```diff
@@ -17,4 +17,4 @@ kubectl apply -f .infrastructure/app/deployment.yml
 
 # Install Ingress Controller
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
-# kubectl apply -f .infrastructure/ingress/ingress.yml
+kubectl apply -f .infrastructure/ingress/ingress.yml
```

Comment:
Ensure that the Ingress resource defined in `ingress.yml` is compatible with the version of the Ingress Controller being installed. Compatibility issues can lead to unexpected behavior.